bogota-july-2018/Calculator-JS/DaniloPlazas/Calculator HTML/scriptJS.js

```diff
@@ -0,0 +1,129 @@
+var num_press = 0.0; 	//Number pressed by the user
+var opt_press = ""; 	//Arithmetic Operation pressed by the user
+var num_acum = 0.0;		//Accumulated Number
+var opt_pend = ""; 		//Arithmetic Operation pending execution
+
+/*
+	FUNCTION: pres_num
+	ARGUMENTS: number (int)
+	RETURN: NA
+*/
+function pres_num(number)
+{
+	var concatenate = "";
+
+	if (num_press === 0) {
+		num_press = number;
+	}else
```

Comment:
Minor: Talking about coding style `{` should be on the same line OR not. So place both ones in the same line in this case